foundationdb/src/tuple/item.rs

```diff
@@ -224,7 +228,7 @@ impl Decode for Uuid {
         let mut uuid = [0u8; 16];
         uuid.copy_from_slice(&buf[1..17]);
 
-        Ok((Uuid(uuid), 17))
+        Ok((Uuid::from_uuid_bytes(uuid), 17))
```

Comment:
@yjh0502 I ended up using `from_uuid_bytes` because `from_bytes` doesn't check the first byte like you do.